scripts/gitlab-test.sh

```diff
@@ -44,4 +44,12 @@ case $TEST_SWITCH in
   test-coverage)
     coverage_test
     ;;
+  rustfmt)
+    cargo install rustfmt-nightly
+    cargo fmt -- --write-mode=diff
+    ;;
+  clippy)
+    cargo install clippy
+    cargo clippy -- -D warnings
```

Comment:
same here, right now it will fail with millions of errors, since we don't use it. I think it would be better to first fix the codebase and then introduce this.

Reply:
it is necessary to add in TODO